src/main/webapp/WEB-INF/web.xml

```diff
@@ -7,7 +7,7 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns/javaee/web-app_2_5.xsd"
     version="2.5">
-  <display-name>My Application Seed</display-name>
+  <display-name>Widget Creator</display-name>
```

Comment:
Probably shouldn't be named "Widget Creator". "Your app name here" might be fine.